src/components/graphs/DataByHourGraph/index.tsx

```diff
@@ -200,127 +313,17 @@ function DataByHourGraph({ range, stats = [] }: Props) {
-    }, [intl, stats]);
-
-    // Effect to update the data by updating the series.
-    useEffect(() => {
-        const bytesFormatter = ({ value }: any) => {
-            if (!Number.isInteger(value)) {
-                return intl.formatMessage({
-                    id: 'common.missing',
-                });
-            }
-
-            return defaultDataFormat(value);
-        };
-        const bytesSeries: EChartsOption['series'] = {
-            data: [],
-            markLine: {
-                data: [{ type: 'max', name: 'Max' }],
-                label: {
-                    position: 'start',
-                    formatter: bytesFormatter,
-                },
-                symbolSize: 0,
-            },
-            barMinHeight: 1,
-            name: intl.formatMessage({ id: 'data.data' }),
-            type: 'bar',
-            yAxisIndex: 0,
-            tooltip: {
-                valueFormatter: (value: any) => bytesFormatter({ value }),
-            },
-        };
-
-        const docsFormatter = ({ value }: any) => {
-            if (!Number.isInteger(value)) {
-                return intl.formatMessage({
-                    id: 'common.missing',
-                });
-            }
-
-            return readable(value, 2, false);
-        };
-
-        const docsSeries: EChartsOption['series'] = {
-            barMinHeight: 1,
-            data: [],
-            markLine: {
-                data: [{ type: 'max', name: 'Max' }],
-                label: {
-                    position: 'end',
-                    formatter: docsFormatter,
-                },
-                symbolSize: 0,
-            },
-            name: intl.formatMessage({ id: 'data.docs' }),
-            type: 'bar',
-            yAxisIndex: 1,
-            tooltip: {
-                valueFormatter: (value: any) => docsFormatter({ value }),
-            },
-        };
-
-        // Go through hours so we have data for each hour
-        hours.forEach((hour, index) => {
-            // See if there is data for the hour we're looking for
-            const hourlyDataSet = scopedDataSet[hour];
-
-            // Default to null so we can differentiate between 0 and missing data
-            const bytes = hourlyDataSet?.bytes ?? null;
-            const docs = hourlyDataSet?.docs ?? null;
-
-            // Add custom styling for the last hour as that will be updating dynamically
-            const itemStyle: any = {};
-            if (index === hours.length - 1) {
-                itemStyle.opacity = '0.80';
-            }
-
-            // Add data to series
-            bytesSeries.data?.push({ itemStyle, value: bytes });
-            docsSeries.data?.push({ itemStyle, value: docs });
-        });
-
-        // Update mychart series so new data goes in
-        myChart?.setOption({
-            series: [bytesSeries, docsSeries],
-        });
-    }, [hours, intl, myChart, scopedDataSet]);
-
     return <div id="data-by-hour" style={{ height: 350 }} />;
```

Comment:
This chart container ID should also be stored in an constant for ease of reference (and maintenance).